models/models.js

```diff
@@ -14,4 +14,12 @@ exports.extractArticleById = (article_id) => {
         if (rows.length) return rows[0];
         else return Promise.reject({ errStatus: 404, msg: "Entry not found"});
     });
-};
\ No newline at end of file
+};
+
+exports.updateArticleVotesById = (article_id, body) => {
+        return db.query(`UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *`, [body.inc_votes, article_id])
```

Comment:
So I had to delve deep (far deeper than I'd have liked, tbh) into the murky world of SQL error codes to make this little beauty work, but I assume this is the kinda thing you had in mind, right?

Reply:
Perfect! This is exactly what I was getting at!

Your previous solution seemed to work but in reality it wasn't actually updating the database in any way :)